style.css

```diff
@@ -468,3 +502,242 @@ body {
     grid-template-columns: repeat(3, 1fr);
   }
 }
+
+@media screen and (min-width: 768px) {
+  header {
+    margin: 1.25rem 2.5rem;
+  }
+
+  .logo {
+    margin-left: 155px;
+  }
+
+  p {
+    font-family: 'Poppins', sans-serif;
+    text-align: left;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .homepage {
+    background: url(./images/Aboutme-Big-screen.png) no-repeat;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 12px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    padding: 359px 260px;
+  }
+
+  .intro {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
- [ ] I think some of the designs are broken after the reviews, firstly, please add more padding to this class as it sticks to the top of the 768px screens. 
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/91486791/7d898dc8-4fff-4172-b035-ee65214e7096)